Assets/HotmaxEditorVR/handColliderFollow.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class handColliderFollow : MonoBehaviour
+{
+    //--local refs
+    private stateManager _stateManagerMutatorRef;
+    private bool _selectedObjectIsActive = stateManager.selectedObjectIsActive;
+
+    private GameObject collidedWithHand;
+
+    Transform selectorHand;
```

Comment:
make private, and clean up spacing in initial class vars

Reply:
got it!